drivers/spi/Kconfig.cc13xx_cc26xx

```diff
@@ -10,3 +10,15 @@ config SPI_CC13XX_CC26XX
 	depends on SOC_SERIES_CC13X2_CC26X2
 	help
 	  Enable support for the TI SimpleLink CC13xx / CC26xx SPI peripheral
+
+config SPI_CC13XX_CC26XX_LOOPBACK_MODE_0
+	bool "Enable loopback mode for SPI 0"
+	depends on SPI_0
+	help
+	  Set loopback mode for the TI SimpleLink CC13xx / CC26xx SPI peripheral
+
+config SPI_CC13XX_CC26XX_LOOPBACK_MODE_1
+	bool "Enable loopback mode for SPI 1"
+	depends on SPI_1
+	help
+	  Set loopback mode for the TI SimpleLink CC13xx / CC26xx SPI peripheral
```

Comment:
These 2 sound useless, see below in the code why. 
Also, it is very bad to have per-instance config option in Kconfig now that we are using DTS for configuring driver instances.

Reply:
@tbursztyka Could you clarify? I don't see anything in the code below. It'd help if you can point me to an example where per-instance config option in Kconfig was avoided. In `drivers/spi/Kconfig` I still see the `SPI_0`, `SPI_1` etc. listed as separate config options for each SPI instance.

Reply:
oh, indeed my comment about driver change did not make it... will re-comment.

spi instances in Kconfig are legacy stuff, and we should work on removing them actually.